apps/client/src/app/pages/search/search/search.component.ts

```diff
@@ -77,7 +77,7 @@ export class SearchComponent implements OnInit {
               private rotationPicker: RotationPickerService, private htmlTools: HtmlToolsService,
               private message: NzMessageService, private translate: TranslateService) {
     this.uiCategories$ = this.xivapi.getList(XivapiEndpoint.ItemUICategory, {
-      columns: ['ID', 'Name_de', 'Name_en', 'Name_fr', 'Name_ja'],
+      columns: ['ID', 'Name_de', 'Name_en', 'Name_fr', 'Name_ja', 'Name_zh'],
```

Comment:
Unfortunately, this won't be working because xivapi doesn't provide `Name_zh`, that being said, I should convert this to a basic usage of the itemName pipe inside the template. I'll take care of this part.